server/graphman/tests/deployment_mutation.rs

```diff
@@ -390,3 +390,197 @@ fn graphql_cannot_remove_subgraph_with_invalid_name() {
         assert_ne!(resp, success_resp);
     });
 }
+
+#[test]
+fn graphql_can_unassign_deployments() {
+    run_test(|| async {
+        let deployment_hash = DeploymentHash::new("subgraph_1").unwrap();
+        create_test_subgraph(&deployment_hash, TEST_SUBGRAPH_SCHEMA).await;
+
+        let unassign_req = send_graphql_request(
+            json!({
+                "query": r#"mutation {
+                    deployment {
+                        unassign(deployment: { hash: "subgraph_1" }){
+                            success
+                        }
+                    }
+                }"#
+            }),
+            VALID_TOKEN,
+        )
+        .await;
+
+        let expected_resp = json!({
+            "data": {
+                "deployment": {
+                    "unassign": {
+                        "success": true,
+                    }
+                }
+            }
+        });
+
+        let subgraph_node_id = send_graphql_request(
+            json!({
+                "query": r#"query Deployment {
+                    deployment {
+                        info(deployment: { hash: "subgraph_1" }) {
+                            nodeId
+                        }
+                    }
+                }"#
```

Comment:
Named query does not seem to be necessary.

```suggestion
                "query": r#"{
                    deployment {
                        info(deployment: { hash: "subgraph_1" }) {
                            nodeId
                        }
                    }
                }"#
```